platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncNode.java

```diff
@@ -255,7 +256,7 @@ public ShadowGraphSynchronizer getSynchronizer() throws InterruptedException {
     }
 
     /**
-     * <p>Calls the {@link ShadowGraph#expireBelow(long)} method and saves the {@code expireBelow} value for use in
+     * <p>Calls the {@link ShadowGraph#updateNonExpiredEventWindow(com.swirlds.platform.consensus.NonAncientEventWindow)} method and saves the {@code expireBelow} value for use in
```

Comment:
Nit: line length

Reply:
Auto-formatted this file. Annoying how spotless doesn't format javadocs.